src/pages/AdminManageDul.js

```diff
@@ -205,13 +242,23 @@ class AdminManageDul extends Component {
   dialogHandlerCreate = (answer) => async (e) => {
     this.setState({ disableOkBtn: answer, disableCancelBtn: answer });
     if (answer) {
-      let consentId = this.state.createId;
-      Election.createElection(consentId).then(
+      let dac = this.state.selectedDac;
+      let election = this.state.selectedElection;
+      let consentId = election.consentId;
+      let electionPromise = null;
```

Comment:
Since none of these assignments seem to be changed later, using `const` instead should prevent bugs later on.